build.psm1

```diff
@@ -325,14 +325,6 @@ function Start-PSBuild {
         }
     }
 
-    # save git commit id to file for PowerShell to include in PSVersionTable
-    $gitCommitId = $ReleaseTag
-    if (-not $gitCommitId) {
-        # if ReleaseTag is not specified, use 'git describe' to get the commit id
-        $gitCommitId = Get-PSCommitId -WarningAction SilentlyContinue
-    }
-    $gitCommitId > "$psscriptroot/powershell.version"
-
```

Comment:
Need to remove `powershell.version` from `.gitignore` as well.

Reply:
Fixed.